### PR TITLE
Feature/fix windows configuration

### PR DESCRIPTION
**Pull request by raquel-ucl**
Follow conda convention for Python installation in Windows. Works in my Win7 VM, but can't be tested in the Jenkins Windows nodes until Luke install VS2015.

Related to #64.

### CODE REVIEW

File: cmake_modules/AllPythonBindings.cmake [hunk 1]
```diff
@@ -136,9 +135,9 @@ if(tests)
 endif(tests)
 
 
-
+# Follow conda convention for Python installation in Windows 
 if(WIN32)
-  set(PYINSTALL_DIRECTORY dcprogs/python-pkg)
+  set(PYINSTALL_DIRECTORY ${PYTHON_INTERP_PREFIX}/lib/site-packages)
 else()
```
Comment (jenshnielsen) [inline cmake_modules/AllPythonBindings.cmake]: I don't think this is the right way. PYINSTALL_DIRECTORY should be a relative directory and then CMAKE_INSTALL_PREFIX should be set to ${PYTHON_INTERP_PREFIX} if not explicitly set to something else

**Comment by jenshnielsen** [comment]
I would guess that line 88 and 89 

```
      set(WORKINGDIR ${TEST_INSTALL_ABSPATH}/dcprogs/python-pkg/)
      set(ADD_TO_PATH ${TEST_INSTALL_ABSPATH}/dcprogs/DLLs)
```

also needs to change?

File: CMakeLists.txt [hunk 2]
```diff
@@ -38,6 +37,18 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake_modules)
 include(LookUp-GreatCMakeCookOff)
 include(PackageLookup)
 
+if(pythonBindings)
+  include(${CMAKE_SCRIPTS}/AllPythonBindings.cmake)
+endif(pythonBindings)
+
```
Comment (jenshnielsen) [inline CMakeLists.txt]: I guess the stuff below should be inside the if(pythombindings) too. Otherwise PYTHON_INTERP_PREFIX will likely be empty

Comment (raquel-ucl) [inline CMakeLists.txt]: I haven't thought to that! Yes, done now.

**Comment by raquel-ucl** [comment]
retest this please

**Pull request merged**